### PR TITLE
User add to k8s-infra-gcp-auditors

### PR DESCRIPTION
**Pull request by jimdaga**
## Changes
- Add `jdagostino2@gmail.com` to **k8s-infra-gcp-auditors**
- Add `jdagostino2@gmail.com` to **k8s-infra-gcp-accounting**

## Notes
Paired with @hh today to review how I can help out in the **wg-k8s-infra** group. 
Getting added to **k8s-infra-gcp-auditors** and **k8s-infra-gcp-accounting** group was suggested as a good starting point.
As a start I could help review costs, and possibly suggest areas for improvement. 

Update: Per @hh's comment, adding **k8s-infra-gcp-accounting**

### CODE REVIEW

**Comment by jimdaga** [comment]
Contributor License Agreement has been signed

**Comment by hh** [comment]
I think you might want to take a look at `k8s-infra-gcp-accounting` as well.
/lgtm

File: groups/wg-k8s-infra/groups.yaml [hunk 1]
```diff
@@ -111,6 +111,7 @@ groups:
       - spiffxp@gmail.com
       - spiffxp@google.com
       - thockin@google.com
+      - jdagostino2@gmail.com
```
Comment (ameukam) [inline groups/wg-k8s-infra/groups.yaml:114]: @jimdaga 👋🏾  can you keep this list ordered alphabetically ?

File: groups/wg-k8s-infra/groups.yaml [hunk 2]
```diff
@@ -130,6 +131,7 @@ groups:
       - spiffxp@google.com
       - spiffxp@gmail.com
       - thockin@google.com
+      - jdagostino2@gmail.com
```
Comment (ameukam) [inline groups/wg-k8s-infra/groups.yaml:134]: @jimdaga same here.

File: groups/wg-k8s-infra/groups.yaml (hunk 1 above)
Comment (jimdaga) [reply groups/wg-k8s-infra/groups.yaml]: @ameukam  I have corrected this (also fixed one existing name that was out of order).

File: groups/wg-k8s-infra/groups.yaml (hunk 2 above)
Comment (jimdaga) [reply groups/wg-k8s-infra/groups.yaml]: Fixed

**Comment by ameukam** [comment]
/lgtm
/hold

/assign @dims @spiffxp 
Leave the approval to them.

**Review by spiffxp — APPROVED** [review]
/hold cancel
/approve
/lgtm

**Pull request merged**